contrib-prompt/functions/prompt_sorin-contrib_setup

```diff
@@ -0,0 +1,132 @@
+# An extension of the existing sorin theme to add some contrib features.
+#
+# Authors:
+#   Jonathan Pigrée <jonathan.pigree@gmail.com>
+#
+# Screenshots:
+#   http://i.imgur.com/nrGV6pg.png
+#
+
+#
+# 16 Terminal Colors
+# -- ---------------
+#  0 black
+#  1 red
+#  2 green
+#  3 yellow
+#  4 blue
+#  5 magenta
+#  6 cyan
+#  7 white
+#  8 bright black
+#  9 bright red
+# 10 bright green
+# 11 bright yellow
+# 12 bright blue
+# 13 bright magenta
+# 14 bright cyan
+# 15 bright white
+#
+
+# Load dependencies.
+pmodload 'helper'
+
+function load_sorin {
+    local found=0
+    for fp in $fpath
+    do
+        if [[ $fp =~ "/prompt/functions" ]]
+        then
+            found=1
+            source $fp/prompt_sorin_setup
+            break
+        fi
+    done
+
+    if [ $found -eq 0 ]
+    then
+        echo "Couldn't found sorin prompt in loaded function files."
+    fi
+}
+
+function prompt_sorin_contrib_async_callback {
+  case $1 in
+    prompt_sorin_contrib_async_k8s)
+      read _k8s_cluster _k8s_ns <<<"$3"
+
+      if [ -z "$_k8s_cluster" ]
```

Comment:
Please prefer `[[` and `]]` over the single bracketed versions.

Reply:
Okay